virtualization/windowscontainers/docker/configure_docker_daemon.md

```diff
@@ -31,19 +31,19 @@ Download the Docker Engine
 
 The latest version may always be found at https://master.dockerproject.org . This sample uses the latest from the v1.13-development branch. 
 
-```none
+```powershell
 Invoke-WebRequest "https://download.docker.com/components/engine/windows-server/cs-1.12/docker.zip" -OutFile "$env:TEMP\docker.zip" -UseBasicParsing
```

Comment:
The cs builds are intended for Windows Server 2016 where Docker commercial support is available. 

At this point, [v1.13 is in code freeze](https://github.com/docker/docker/wiki) and should be working on Windows 10. If it's failing, please be sure to get an issue filed at https://github.com/docker/docker/issues